src/features/events/models/EventsModel.ts

```diff
@@ -0,0 +1,55 @@
+import Environment from 'core/env/Environment';
+import EventsRepo from '../repo/EventsRepo';
+import { ModelBase } from 'core/models';
+import { ZetkinEvent } from 'utils/types/zetkin';
+import {
+  ErrorFuture,
+  IFuture,
+  LoadingFuture,
+  ResolvedFuture,
+} from 'core/caching/futures';
+import { isAfter, isBefore, isSameDate } from 'utils/dateUtils';
+
+export class EventsModel extends ModelBase {
+  private _orgId: number;
+  private _repo: EventsRepo;
+
+  constructor(env: Environment, orgId: number) {
+    super();
+    this._orgId = orgId;
+    this._repo = new EventsRepo(env);
+  }
+
+  getAllEvents(): IFuture<ZetkinEvent[]> {
+    return this._repo.getAllEvents(this._orgId);
+  }
+
+  getEventsInTimespan(
```

Comment:
This function does exactly what it needs to do as far as I can tell, but the name implies something else to me.

The name implies to me that it find events that take place no earlier than `startString` and no later than `endString`. But what it really does is find events take place on or after the _date_ of `startString` (ignoring time) and on or before the _date_ of `endString` (ignoring time).

Again, this is correct, but the name implies something else to me.

Reply:
getParallelEvents ?